configs/server/forseti_conf_server.yaml.in

```diff
@@ -211,7 +211,7 @@ scanner:
     # gs://bucket-name/path/for/rules_path
     # if no rules_path is specified, rules are
     # searched in /path/to/forseti_security/rules/
-    rules_path: /home/ubuntu/forseti-security/rules
+    rules_path: gs://{FORSETI_BUCKET}/rules
```

Comment:
I am wondering how this will impact the current behavior on the VM.

We currently have the VM's startup script to pull down the rule files from GCS on every startup, and after that Forseti will read the rule files locally.  

https://github.com/forseti-security/forseti-security/blob/162e23039dcf399b18dcb57e266d1987f2f1bc2a/deployment-templates/compute-engine/server/forseti-instance-server.py#L243

Should we instead make the rules_path to be a user-configurable deployment value?  With a default for local  path?

Reply:
Unfortunately that will not work in a container.  Just out of curiosity, what is the downside to having the VM read the rules from GCS by default?

Reply:
There might be a way in the helm charts to replace the value of `rules_path` on load.  Lemme think on it.

Reply:
@kevensen Were you able to find more info about this? 

Wondering if Henry's suggestion to actually make the `rules_path` a separate variable with the default being `/home/ubuntu/forseti-security/rules` is the better, more scalable option, as that would let GKE users just configure it from the get go and not have to redeploy.

Reply:
That's fine by me.  I think that the fix via the Helm chart is sufficient.  Thoughts?